modules/k8s/client.go

```diff
@@ -34,6 +34,9 @@ func GetKubernetesClientFromOptionsE(t testing.TestingT, options *KubectlOptions
 			return nil, err
 		}
 		logger.Log(t, "Configuring Kubernetes client to use the in-cluster serviceaccount token")
+	} else if options.RestConfig != nil {
+		config = options.RestConfig
+		logger.Log(t, "Configuring Kubernetes client to use provided rest config object")
```

Comment:
Will be helpful to log which config file was used, which will help in future debugging

Reply:
Thanks for reply. However, I'm sorry, I think I don't understand what exactly are you suggesting to log.

RestConfig don't use the kubeconfig file, but instead, it uses direct credentials (API address, cert file, etc..) from a provided object as you can see in the description.

__Note:__ _RestConfig can be generated also from the kubeconfig file the same as I did it in the test. But goal of the PR is that it does not matter where the config comes from._

Maybe I could improve the log by providing atleast a kube API address. Will check what can be useful here as there are more parameters.

Reply:
Added server address to log as it seems the only attr that should be always present.